swift/example-qr-generator/QrTests/QrTests.swift

```diff
@@ -0,0 +1,35 @@
+//
```

Comment:
I assume this tests are for sample app. Its sample so no need to have tests. Can we remove it?

Reply:
done